markdown-scheme_v3.Rmd

```diff
@@ -0,0 +1,393 @@
+# Project description
+## Title statement
+Sufficient fields are
+\begin{enumerate}
+  \item Primary ID: Ideally a DOI or an organization consistent scheme;
+      \begin{enumerate}
+        \item Published Paper: 
+            \begin{enumerate}
+              \item A corresponding PRWP exists: FORMAT
+                \begin{lstlisting}
+                  PP_CountryCodeIso3_Year_PRWP-PRWPno_versionno
+                  Example: PP_CRI_2018_PRWP-8524_v01
+                \end{lstlisting}
+              \item A corresponding PRWP does not exit: FORMAT
+                \begin{lstlisting}
+                  PP_CountryCodeIso3_Year_EXT-EXTno_versionno
+                  Example: PP_CRI_2018_EXT-1_v01
+                \end{lstlisting}
+                    EXTno is a unique running number assigned by Dime.
+            \end{enumerate}
+        \item Working Paper: FORMAT
+          \begin{lstlisting}
+            RR_CountryCodeIso3_Year_PRWP-PRWPno_versionno
+            RR_CRI_2018_PRWP-8525_v01
+          \end{lstlisting}
+        \end{enumerate}
+  \item Other identifiers: DOI and https://doi.org/number
+  \item Title: Of the following form
+    \begin{lstlisting}
+      Reproducibility package for TITEL(paper)
+    \end{lstlisting}
+  \item Subtitle:
+    \begin{itemize}
+      \item WP: World Bank Policy Research Working Paper (PRWP) number
+      \item PP: DOI number (no https)
+    \end{itemize}
+\end{enumerate}
+
+## Authors and contributors
+Sufficient are
+\begin{enumerate}
+    \item Authoring entity: author, affiliation and ORCID number and email when available; For world bank authors use \textbf{World Bank}. Example:
+      \begin{verbatim}
+        Christoph Lakner World Bank ORCID 0000, clakner@worldbank.org
+      \end{verbatim}
+    \item Contributors: like RAs (not conference participants). Also coeditors if specifically mentioned. FORMAT
+    \begin{lstlisting}
+      Coeditor
+    \end{lstlisting}
+    \item Sponsors: All financial contributions and individual/general grants
+    \item Acknowledgements: Usually empty; Example: Corporate Taxation: special mentioning ministry employees for data assistance (could go to acknowledgment statement)
+    \item Acknowledgement Statement:
+      \begin{verbatim}
+        We are grateful to Benu Bidani, Francisco Ferreira, Luis-Felipe..
+      \end{verbatim}
+      \begin{verbatim}
+        .. and seminar participants at the World Bank’s Poverty ...
+      \end{verbatim}
+  \end{enumerate}
+
+## Version statement
+Sufficient are
+\begin{enumerate}
+    \item Project completion date: FORMAT: YEAR-MONTH
+      \begin{itemize}
+        \item PP: Date at which the article was published: Year-Month
+        \item WP: \textcolor{red}{Must be supplied, bc it is needed for the front page
+        of cataloge}
+      \end{itemize}
+    \item Version date: FORMAT: YEAR-MONTH-DAY
+      \begin{itemize}
+        \item PP: empty
+        \item WP: data of publication as research paper in WB Year-Month-Day
```

Comment:
not clear to me how this is different from the 'project completion date' explained above. when would these be different? make a clearer case for how the information is distinct or drop one.